### PR TITLE
Assignment 6.1 Overview

### PR DESCRIPTION
**Pull request by alkreb**
Rewrite for To-do API overview page.

### CODE REVIEW

File: docs/index.md
Comment (rbwatson) [inline docs/index.md:1]: Moved link to new section.

File: docs/overview-alicia.md
Comment (rbwatson) [inline docs/overview-alicia.md:1]: Looks good!

**Comment by rbwatson** [comment]
1. Rebased PR branch to include new main branch changes.
2. Moved link to new pages on landing page in main branch.
3. Squashed commits.

File: docs/overview-alicia.md
Comment (alkreb) [reply docs/overview-alicia.md:1]: Thank you!

**Pull request merged**